exercises/ex1.cpp

```diff
@@ -11,38 +14,39 @@ int main()
   cout << "Enter week number(1-7): " << endl;
   cin >> week;
 
-  if (week == 1)
-  {
-    cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
-    cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
-    cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
-    cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
-    cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
-    cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
-    cout << "Sunday" << endl;
-  }
-  else
-  {
-    cout << "Invalid input! Please enter week number between 1-7." << endl;
+  switch (week) {
+
+    case 1:
+      cout << "Monday";
+      break;
+
+    case 2:
+      cout << "Tuesday";
+      break;
+
+    case 3:
+      cout << "Wednesday";
+      break;
+
+    case 4:
+      cout << "Thursday";
+      break;
+
+    case 5:
+      cout << "Friday";
+      break;
+
+    case 6:
+      cout << "Saturday";
+      break;
+
+    case 7:
+      cout << "Sunday";
+      break;
+
+    default:
+      cout << "\nInvalid input! Please enter week number between 1-7.\n";
```

Comment:
non hai messo il cout << endl dopo lo switch case, comunque non è rilevante in realtà